### ISSUE TITLE
Review Helm System Charts in Rancher for k8s 1.22 support

### ISSUE THREAD

**Issue by brendarearden** [rancher/rancher#35897]
### Related issue
https://github.com/rancher/rancher/issues/34125

This issue is used to track helm system charts within Rancher to put kubeversion limits

**Comment by jiaqiluo** [comment]
### Root cause
The `kubeVersion` filed is missing in the Chart.yaml in some charts. As the result, a chart that does not support 1.22 might show in a 1.22 cluster, and of course, the installation will fail. 

### What was fixed, or what changes have occurred
add `kubeVersion` to charts with proper values. 

### Areas or cases that should be tested
make sure the proper version of the charts show in the cluster:
- `rancher-istio 1.5.920` and `rancher-monitoring 0.1.5 & 0.3.1` is available on cluster which k8s < 1.21 
- `rancher-cis-benchmark 0.2.0` and `rancher-external-dns 0.2.0`  is available on cluster which k8s < 1.22
- `rancher-k3s-upgrader 0.3.1` is available on cluster with k8s >= 1.16
- `rancher-logging 0.4.0`  is available on cluster with k8s >= 1.21 


### What areas could experience regressions?
n/a

### Are the repro steps accurate/minimal?
yes.

**Comment by anupama2501** [comment]
Verified the charts annotations:

monitoring 0.3.1 and 0.1.5 
```
kubeVersion: "< 1.22.0-0"
```

Logging 0.4.0
```
kubeVersion: ">=1.21.0-0"
```

istio 1.5.920
```
kubeVersion: '< 1.21.0-0'
```

k3s upgrader 0.3.1
```
kubeVersion: '>= 1.16.0-0'
```

DNS 0.2.0
```
kubeVersion: '< 1.22.0-0'
```

cis benchmark 0.2.0
```
kubeVersion: '< 1.22.0-0'
```
